builtin/commit.c

```diff
@@ -56,19 +56,19 @@ static const char empty_cherry_pick_advice[] =
 N_("The previous cherry-pick is now empty, possibly due to conflict resolution.\n"
 "If you wish to commit it anyway, use:\n"
 "\n"
-"    git commit --allow-empty\n"
-"\n");
+"    git commit --allow-empty\n");
 
 static const char empty_cherry_pick_advice_single[] =
-N_("Otherwise, please use 'git reset'\n");
+N_("\nOtherwise, please use 'git cherry-pick --skip'\n");
 
 static const char empty_cherry_pick_advice_multi[] =
-N_("If you wish to skip this commit, use:\n"
+N_("    git cherry-pick --continue\n\n"
+"to resume cherry-picking the remaining commits.\n"
```

Comment:
So we'd end up with a message like:

If you wish to commit anyway, use:
    git commit --allow-empty
    git cherry-pick --continue
to resume cherry-picking the remaining commits.

That makes it sound at first like the cherry-pick --continue is part of the command to commit the empty commit anyway, and then causes a double take once you hit "to resume cherry-picking".  It'd be nice if there were some words between the two commands, e.g. "and then run:"

Reply:
> That makes it sound at first like the cherry-pick --continue is part of the command to commit the empty commit anyway

Yes, it could indeed lure users in believing that.